python/hidet/ir/primitives/cpu/avx.py

```diff
@@ -69,6 +87,51 @@ def register_primitive_functions():
     for name, codegen_name, func_type in functions:
         register_primitive_function(name=name, func_or_type=func_type, codegen_name=codegen_name)
 
+    from hidet.lang import script, attrs
+    from hidet.ir.dtypes import f32x8, f32
+    from hidet.ir.func import Function
+
+    @script
+    def avx_x86_f32x8_sum(x: f32x8) -> f32:
```

Comment:
Would it be possible to only declare the primitives (e.g., `avx_x86_f32x8_extract_half`) in this file, and then define functions like `avx_x86_f32x8_sum` as helper functions in Hidet Script in a separate file where it would be needed? The code should work as it is, but it looks a bit odd to have `hidet.script`decorator and multiple calls to `call_primitive_func` here...